libr/core/cmd_debug.c

```diff
@@ -1651,6 +1651,20 @@ static int cmd_debug_map(RCore *core, const char *input) {
 					if (libname) {
 						char *cmd, *res;
 						const char *file = map->file? map->file: map->name;
+						char *newfile = NULL;
+						if (!r_file_exists (file)) {
+							newfile = r_file_temp ("memlib");
+							if (newfile) {
+#if __WINDOWS__ && !__CYGWIN__
+								const char *nul = "nul";
+#else
+								const char *nul = "/dev/null";
+#endif
+								file = newfile;
+								r_core_cmdf (core, "wtf %s 0x%"PFMT64x" @ 0x%"PFMT64x" 2> %s",
```

Comment:
i would rather prefer to have an r_sys_devnull() function that returns the right path to the null device

Reply:
okay